lib/Backends/Habana/Habana.cpp

```diff
@@ -489,22 +541,41 @@ llvm::Error HabanaFunction::execute(ExecutionContext *context) {
     RETURN_ERR_IF_NOT(T, "Failed to get input tensor.");
 
     bool isPartial = partialInputs_.count(P);
+    bool isInt64 = int64Inputs_.count(P);
```

Comment:
nit: `isInt64` could be named better -- something can be int64 but return false for `allows64To32Downcast()`, right? Maybe `convertInt64To32`